NAPS2.Lib.WinForms/EtoForms/WinForms/WinFormsListView.cs

```diff
@@ -26,6 +28,8 @@ public WinFormsListView(ListViewBehavior<T> behavior)
         _behavior = behavior;
         _view = behavior.ScrollOnDrag ? new DragScrollListView() : new ListView();
         _view.MultiSelect = behavior.MultiSelect;
+
+        _config = Naps2Config.Stub();
```

Comment:
"Stub" config is only for testing, it won't actually work. What I recommend:
- Add Naps2Config to the ImageListViewBehavior constructor, this will get the right object automatically through Dependency Injection
- Use that to set a new `ShowPageNumbers` property on ListViewBehavior
- Use `_behavior.ShowPageNumbers` in this class

Reply:
Humm. You mean, I add Naps2Config in the variables of the class definition? I'm not really an advanced programmer in c#...
Can you give me more details? This elude me a bit...

Reply:
[Here](https://github.com/cyanfish/naps2/blob/6f4787464b4da1e7eb5d3a2259fab0a955eed40d/NAPS2.Lib/EtoForms/Widgets/ImageListViewBehavior.cs#L13) you can see some existing parameters (UiThumbnailProvider, ImageTransfer, ColorScheme), you can add one more for `Naps2Config config`, and then `ShowPageNumbers = config.Get(c => c.ShowPageNumbers)` [here](https://github.com/cyanfish/naps2/blob/6f4787464b4da1e7eb5d3a2259fab0a955eed40d/NAPS2.Lib/EtoForms/Widgets/ImageListViewBehavior.cs#L23) (after creating the ShowPageNumbers property).

This is a kind-of-advanced programming concept called [Dependency Injection (DI)](https://stackoverflow.com/questions/130794/what-is-dependency-injection) or Inversion of Control (IoC) where, to put it very simply, when you add a parameter to the constructor like that it "automatically" sets it to the correct object (as configured [elsewhere](https://github.com/cyanfish/naps2/blob/6f4787464b4da1e7eb5d3a2259fab0a955eed40d/NAPS2.Lib/Modules/CommonModule.cs#L47) in the code).